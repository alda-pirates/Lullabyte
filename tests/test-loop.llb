void main() {
	sound c;
	sound d;
	sound e;
	sound scale;
	sound s;
	
	c = |C1|:0.25:100;
	d = |D1|:0.25:100;
	e = |E1|:0.25:100;
	
	scale =	[c, d, e];
	print(scale);
	loop(s : scale) {
		s = s * 3;
	}
	print(scale);
}