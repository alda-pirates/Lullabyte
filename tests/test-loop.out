[|C1|:0.25:100, |D1|:0.25:100, |E1|:0.25:100]
[|C1|:0.75:100, |D1|:0.75:100, |E1|:0.75:100]